internal/schema/ize-spec.json

```diff
@@ -0,0 +1,114 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+        "aws_profile": {
+            "type": "string"
+        },
+        "aws_region": {
+            "type": "string"
+        },
+        "config_file": {
+            "type": "string"
+        },
+        "env": {
+            "type": "string"
+        },
+        "env_dir": {
+            "type": "string"
+        },
+        "home": {
+            "type": "string"
+        },
+        "infra_dir": {
+            "type": "string"
+        },
+        "log_level": {
+            "type": "string"
+        },
+        "namespace": {
+            "type": "string"
+        },
+        "plain_text": {
+            "type": "boolean"
+        },
+        "prefer_runtime": {
+            "type": "string"
+        },
+        "projects_path": {
```

Comment:
That's the path I'm referring to, it's quite confusing to have the repo as a project and projects path. Maybe this should be come "apps_path"?